.github/workflows/gradle.yml

```
@@ -0,0 +1,43 @@
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Set up JDK 11
+      uses: actions/setup-java@v4
+      with:
+        distribution: 'temurin'
+        java-version: '11'
+        cache: 'gradle'
+
+    - name: Cache ZNPCs jar
+      uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)